### PR TITLE
legoland/ragekick

### PR DESCRIPTION
**Pull request by pixelranchllc**

### CODE REVIEW

**Comment by pixelranchllc** [comment]
RageKick Form Refactor

https://app.clickup.com/t/1cwdrm1
- Refactor ragekick transaction to new service pattern
- Refactors profileMenu items to use openFormModal
- Removes GenericModals for profileMenu menuItems from Profile.js
- Includes changes from legoloand/ragequit

File: src/components/profileMenu.jsx [hunk 1]
```diff
@@ -12,31 +12,49 @@ import {
...
+      lego: {
+        ...FORM.GUILDKICK,
+        localValues: { memberAddress: member.memberAddress },
+      },
+    });
+  };
+
+  const handleRageQuitClick = () => {
+    openFormModal({
+      lego: {
```
Comment (skuhlmann) [inline src/components/profileMenu.jsx]: no need to create a new object on this or the next one - only need to do that if we are overriding a lego property or adding localValues like in guildkick.

```
    lego: CORE_FORMS.RAGE_QUIT
```

**Review by skuhlmann — COMMENTED** [review]
Looking good. one small change.

**Comment by skuhlmann** [comment]
Also looks like we have a conflict to resolve :point_up:

**Comment by pixelranchllc** [comment]
Change made and conflicts resolved.

**Pull request merged**